kubernetes/env-create.md

```diff
@@ -0,0 +1,232 @@
+---
```

Comment:
Just realised that we've got the same doc twice under different paths, so we should lose this one